Add some text to version string in main menu to indicate
that this is a modified version.

diff --git a/Source/mainmenu.cpp b/Source/mainmenu.cpp
index 52b50690..b2dd86fc 100644
--- a/Source/mainmenu.cpp
+++ b/Source/mainmenu.cpp
@@ -92,7 +92,7 @@ void mainmenu_loop()
 
 	do {
 		menu = 0;
-		if (!UiMainMenuDialog("Diablo v1.09", &menu, effects_play_sound, 30))
+		if (!UiMainMenuDialog("Diablo v1.09 with QOL changes", &menu, effects_play_sound, 30))
 			app_fatal("Unable to display mainmenu");
 
 		switch (menu) {
